fescpw/src/redux/actions/index.js

```diff
@@ -0,0 +1,29 @@
+export { 
+    SRH_EDT,
+    srhEdt
+} from './srhActions';
+export {
+    NTS_FTH_IP, 
+    NTS_FTH_SCS, 
+    NTS_FTH_FLR,
+    NTS_SGR_IP,
+    NTS_SGR_SCS,
+    NTS_SGR_FLR,
+    NTS_CRT_IP,
+    NTS_CRT_SCS,
+    NTS_CRT_FLR,
+    NTS_GPE_IP,
+    NTS_EDT_IP,
+    NTS_EDT_SCS,
+    NTS_EDT_FLR,
+    NTS_RMV_IP,
+    NTS_RMV_SCS,
+    NTS_RMV_FLR,
+    
```

Comment:
I like the consistency of your naming convention, but maybe put a guide or key somewhere in the code. The readme, or commented somewhere. Just looking at these it's hard to discern what is what.